sharding/nodesCoordinator/indexHashedNodesCoordinator.go

```diff
@@ -611,7 +614,13 @@ func (ihnc *indexHashedNodesCoordinator) EpochStartPrepare(metaHdr data.HeaderHa
 	unStakeLeavingList := ihnc.createSortedListFromMap(newNodesConfig.leavingMap)
 	additionalLeavingList := ihnc.createSortedListFromMap(additionalLeavingMap)
 
+	chainParamsForEpoch, err := ihnc.chainParametersHandler.ChainParametersForEpoch(newEpoch)
+	if err != nil {
+		log.Warn("indexHashedNodesCoordinator.EpochStartPrepare: could not compute chain params for epoch", "epoch", newEpoch, "error", err)
```

Comment:
maybe add the message here, on L1057 and in sharding/nodesCoordinator/indexHashedNodesCoordinatorLite.go : "... and will use the current chain parameters" as that is the fallback.

Reply:
added